analysis/filter/EdgeFilter/src/main/kotlin/de/maibornwolff/codecharta/filter/edgefilter/ParserDialog.kt

```diff
@@ -0,0 +1,38 @@
+package de.maibornwolff.codecharta.filter.edgefilter
+
+import com.github.kinquirer.KInquirer
+import com.github.kinquirer.components.promptInput
+import de.maibornwolff.codecharta.tools.interactivecli.ParserDialogInterface
+import picocli.CommandLine
+
+class ParserDialog {
+    companion object : ParserDialogInterface {
+        private const val EXTENSION = "cc.json"
+        override fun generateParserArgs(args: Array<String>, commandLine: CommandLine): Array<String> {
+
+            var inputFileName = KInquirer.promptInput(message = "What is the $EXTENSION file that has to be parsed?")
+            if (inputFileName.substringAfter(".") != EXTENSION)
+                inputFileName += ".$EXTENSION"
+            println("File path: $inputFileName")
```

Comment:
Not sure, if we should use println when using picocli. If I remember correctly, we use a logger to log to fixed logstreams like info or error.

Reply:
Thank you for the tip. Indeed I found Kotlin Logger being used, and I changed the println to logger.info.